packages/auth/src/core/auth/auth_impl.ts

```diff
@@ -422,6 +427,42 @@ export class AuthImpl implements AuthInternal, _FirebaseService {
     }
   }
 
+  _getPasswordPolicy(): PasswordPolicy | null {
+    if (this.tenantId === null) {
+      return this._projectPasswordPolicy;
+    } else {
+      return this._tenantPasswordPolicies[this.tenantId];
+    }
+  }
+
+  async _updatePasswordPolicy(): Promise<void> {
+    const response = await _getPasswordPolicy(this);
+
+    // Check that the policy schema version is supported by the SDK.
```

Comment:
nit: also add a comment like // TODO - Update this logic to use a MAX_SUPPORTED_PASSWORD_POLICY_SCHEMA_VERSION once we have multiple schema versions.

Reply:
Done.